hw3/01-routing.js

```diff
@@ -34,18 +31,72 @@ const server = http.createServer((req, res) => {
     return result;
   };
 
-  if (req.url === '/') {
+
+  if (req.url === `/`) {
```

Comment:
The way we parse the URL request would be a bit different in Express. You can find examples of it easily.